### ISSUE TITLE
Question about training Pascal VOC

### ISSUE THREAD

**Issue by Womcos** [zhanghang1989/PyTorch-Encoding#329]
Very great repository. Here i have some questions about the training process of Pascal VOC. When i check the train_aug dataset prepared by the prepare_pascal.py script, i notice that the data split way is different to other widely used method, e.g. deeplab (If the split ways are same, the following content can be ignored). For example, the train split data includes some data from the official val set. And here is my question, have you ever compared the evaluation result of two train_aug split ways (the split ways used in this repository and deeplab: tensorflow/models/research/deeplab) on the Pascal VOC test set, since the latter one is convient for conducting ablation experiments on the val set, while the formal one can only evaluate models by submitting test results to the official website. In addition, if we use the latter split way to conduct our experiments, what's the training protocol: training epochs and learning rates for pascal_aug and pascal_voc data.
